xbmc/playlists/PlayList.h

```diff
@@ -14,6 +14,11 @@
 #include <string>
 #include <vector>
 
+constexpr int PLAYLIST_NONE = -1;
```

Comment:
```c++
namespace
{

}
```

Reply:
anonymous namespaces in headers are discouraged.
i think in this particular case it would not be an issue since the constexpr's are evaluated at compiletime anyway.
still i would leave it as is to avoid the anti-pattern.

ref. https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file